config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration.h

```diff
@@ -1245,7 +1236,7 @@
 //#define AUTO_BED_LEVELING_LINEAR
 //#define AUTO_BED_LEVELING_BILINEAR
 //#define AUTO_BED_LEVELING_UBL
-#define MESH_BED_LEVELING
+//#define MESH_BED_LEVELING
```

Comment:
This is for restoring default value, because the parameter has no meaning without BLtouch (forgotten).